messaging/src/main/java/io/lantern/messaging/Messaging.kt

```diff
@@ -886,10 +887,16 @@ class Messaging(
     }
 
     internal fun notifyWebRTCSignal(
+        sent: Long,
         senderId: String,
         senderDeviceId: String,
         content: ByteString
     ) {
+        val delta = now - sent
+        if (delta > webRTCSignalingTimeoutMillis || delta < -1 * webRTCSignalingTimeoutMillis) {
```

Comment:
Does this mean if you are on a call and I try calling you, we will drop my signal? (since we are not distinguishing wtr to senderId)

Reply:
Yes, that is correct. messaging-android doesn't know when you're in a call or in general what's being done with the signaling messages, it just transports them.